returnn/tf/layers/basic.py

```diff
@@ -4769,39 +4773,57 @@ def __init__(self, eval, **kwargs):
 
 class CompareLayer(LayerBase):
   """
-  This layer performs an elementwise comparison of all provided sources, and returnns a boolean vector
-  of the same shape.
-  If the `value` parameter is provided, all elements in the provided sources are also compared to this value.
+  Compares element-wise the tokens of all input sequences among themselves and/or with a specified given value.
+  The comparisons are performed in a chain according to the order in which they are listed. 
 
-  If more than two sources are provided, each additional source will be compared against the first source,
-  and the results will be combined by a logical conjunction.
+  Example:
```

Comment:
Note: You should follow [rst syntax](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html) here, because we use this for the documentation.